src/main/java/com/hyperlocal/server/Merchant.java

```diff
@@ -0,0 +1,47 @@
+package com.hyperlocal.server;
+
+import com.github.jasync.sql.db.RowData;
+import com.google.gson.JsonObject;
+
+public class Merchant {
```

Comment:
We can use Autovalue here which will autogenerate these value classes, getters and setters can be autogenerated.

https://www.baeldung.com/introduction-to-autovalue

Reply:
Added to TODO. Will read this up and implement